internal/provider/metric_export_prometheus_config_resource_test.go

```diff
@@ -0,0 +1,213 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"github.com/cockroachdb/cockroach-cloud-sdk-go/pkg/client"
+	mock_client "github.com/cockroachdb/terraform-provider-cockroach/mock"
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"log"
+	"net/http"
+	"os"
+	"testing"
+)
+
+// TestAccMetricExportPrometheusConfigResource attempts to create, check, and destroy
+// a real cluster. It will be skipped if TF_ACC isn't set.
+func TestAccMetricExportPrometheusConfigResource(t *testing.T) {
+	t.Skip("Skipping until we can either integrate the AWS provider " +
+		"or import a permanent test fixture.")
```

Comment:
Does this rely on an AWS resource?  It seems like it wouldn't....

Reply:
I think this block was copied from other metric export integrations. This is suppose to spin up an actual AWS based CRDB cluster and run the tests on that. To test the interaction with the cloud providers.  But this seems to be `t.Skip()`d in all other integrations. So, it's better to just remove this block until we actually plan to run this test in the future.

Reply:
In general I prefer to keep the skip blocks so that when the tests run, we can see which tests are skipped.  But considering that the retry logic in this resource is fairly complex and that it does not rely on something in the AWS provider, we should add an acceptance test for it.

If you look at another resource where this is implemented you can see that the addition of the acceptance test is much simpler than the integration test since no mocking needs to occur.   Usually it looks something like [this](https://github.com/cockroachdb/terraform-provider-cockroach/blob/main/internal/provider/maintenance_window_test.go#L34-L38).  Let me know if you need further help adding this.

Reply:
We have added this back and removed `t.Skip()`